automations/wet-bulb/input.md

```diff
@@ -0,0 +1,37 @@
+# Wet Bulb Calculator
+
+Welcome to the Wet Bulb Calculator, a user-friendly tool built on CrossCompute. It quickly calculates the wet bulb temperature from your input of ambient temperature and relative humidity. This data plays a vital role in meteorology, HVAC design, and environmental studies. Experience seamless and informative calculations with our calculator—simplifying complex meteorological computations and empowering your understanding of atmospheric conditions. Start today!
+
+{ temperature }
+{ relative_humidity }
+{ BUTTON_PANEL }
+
+## Instructions for Using the Wet Bulb Calculator
+Our tool is designed with simplicity and usability in mind. Let's walk through how to use it:
+
+### Provide the Inputs
+On the main screen of the Wet Bulb Calculator, you'll find two input fields: 'Temperature' and 'Relative Humidity'.
+
+- Temperature: Enter the ambient or 'dry bulb' temperature here. Please note, the unit of temperature isn't specified in the tool, so ensure you maintain consistency in the unit you choose (either degrees Celsius or Fahrenheit) for accurate calculations.
+
+- Relative Humidity: Enter the relative humidity in this field. It's a measure of the amount of moisture in the air compared to the maximum amount the air could hold at the given temperature, expressed as a percentage.
+
+### Submit your Inputs
+Once you've entered the values, click on the 'Continue' button. Our tool will process the inputs, running calculations in the back-end to determine the wet bulb temperature.
+
+### Understand the Output
+In the next page the tool will return the wet bulb temperature.
+
+The wet bulb temperature is the lowest temperature that can be reached by the evaporation of water only. It's a key parameter used in many applications, such as meteorology, HVAC system design, and more. Again, the unit of this output will be consistent with the unit you used for the input temperature.
+
+## Understanding the Wet Bulb Calculator Outputs
```

Comment:
I really like this explanation on the significance of the wet bulb measurement. It adds context to why a concept is important and adds educational value